src/modules/auth/auth-service/auth.service.ts

```diff
@@ -44,4 +45,14 @@ export class AuthService implements AuthServiceInterface {
     async validateUserByCridentials(userLogin: UserLoginRequest): Promise<[Error, UserDbModel]> {
         return await this.userPersistanceService.authenticateUser(userLogin);
     }
+
+    /* istanbul ignore next */
+    static getUserProfileFromToken(token: string): UserTokenProfile {
+        let user: UserTokenProfile = defaultUserTokenProfile;
```

Comment:
Why do we need defaultUserToken? And if so, we could just create empty instance of `UserTokenProfile` instead of createing empty object in `user-token-profile.model.ts`

Reply:
done